go/ct/driver/run.go

```diff
@@ -97,6 +97,14 @@ func doRun(context *cli.Context) error {
 	}
 
 	opRun := func(state *st.State) rlz.ConsumerResult {
+		result := rlz.ConsumeContinue
+		defer func() {
+			if r := recover(); r != nil {
+				result = rlz.ConsumeAbort
+				issuesCollector.AddIssue(state, fmt.Errorf("failed to process input state %v: %w", state, err))
+			}
```

Comment:
maybe also at the end of this block you could return result to ensure that no further execution is done ??

Reply:
The execution is already terminated by returning `ConsumeAbort`.

Reply:
The need for `issuesCollector` is that if not used, the program terminates without any output.
I will change the string, to emphasize that the execution panicked.